### PR TITLE
New Barrel HTTP API Cheat Sheet

### PR DESCRIPTION
**Pull request by pjhampton**
https://duck.co/ia/view/barrel_http_cheat_sheet

### CODE REVIEW

**Comment by daxtheduck** [comment]
Metadata placeholder post
            
*If you haven't already, please edit your first comment to include a link to the IA page at the bottom.*
            
---
            
**How to find your IA page**
- New Instant Answer
  -- follow the steps [here](https://duck.co/ia/new_ia) to create your IA page.
              
- Existing Instant Answer
  -- find your IA page from this [list](https://duck.co/ia)

File: share/goodie/cheat_sheets/json/barrel-http-api.json [hunk 1]
```diff
@@ -0,0 +1,44 @@
+{
+    "id": "barrel_http_api_cheat_sheet",
+    "name": "Barrel HTTP API Cheat Sheet",
+    "description": "An overview of the Barrel HTTP API using cURL",
+
+    "metadata": {
+        "sourceName": "Barrel",
+        "sourceUrl" : "https://docs.barrel.org/"
```
Comment (MrChrisW) [inline share/goodie/cheat_sheets/json/barrel-http-api.json]: This should be https://docs.barrel-db.org/reference ?

Comment (pjhampton) [inline share/goodie/cheat_sheets/json/barrel-http-api.json]: Yep! @MrChrisW - thanks for reminding me about this :smiley:

**Comment by zekiel** [comment]
this will redeploy to beta automagically when the naming is updated. 

filename needs to be barrel-http.json
cheat sheet JSON needs to have id: barrel_http_cheat_sheet

**Comment by daxtheduck** [comment]
Automated screenshot from [beta.duckduckgo.com](https://beta.duckduckgo.com/?q=barrel cheat sheet)
                
                [Mobile screenshot](https://ia-screenshots.s3.amazonaws.com/barrel_http_cheat_sheet_mobile.png)

**Comment by tagawa** [comment]
@pjhampton, thanks again for creating the Barrel HTTP API cheat sheet! You’ve been doing a really good job here and we'd like you to help our community tackle the next big challenge - our mission to become the best search engine for programmers: https://duck.co/blog/post/297/help-for-programmers.

When you're ready to move on, please: 

- Create a new Instant Answer to increase our Programming IA coverage

OR

- Test live programming Instant Answers (or those in development) for bugs and suggest improvements

Detailed instructions explaining how you can help are available at the top of this [page](https://github.com/duckduckgo/duckduckgo/wiki/Programming-IA-Coverage).

Thanks again!

**Comment by moollaza** [comment]
@pjhampton Thanks! This LGTM :+1:

**Comment by daxtheduck** [comment]
@pjhampton this is now merged! It should be live on DuckDuckGo within one week (or during the next release) and we'll be sure to notify you when it happens.

After your Instant Answer goes live for the World to use, you will be able to see [traffic information](http://docs.duckduckhack.com/maintaining/tools.html#monitor-your-instant-answer-page) about your IA as well as receive feedback directly from users. 
                
Please refer to the [maintenance guide and docs](http://docs.duckduckhack.com/maintaining/guidelines.html) to keep up with potential changes to your Instant Answer.
                
Thank you again for your help!

**Pull request merged**